libbs/decompilers/ghidra/hooks.py

```diff
@@ -50,22 +51,49 @@ def domainObjectChanged(self, ev):
                 obj = record.getObject()
 
                 if changeType in self.funcEvents:
-                    pass
+                    subType = record.getSubEventType()
+                    if subType == 5:
```

Comment:
Why is this not a Ghidra constant like `self.changeManager.DOCR_SYMBOL_ADDRESS_CHANGED`?